packages/frontend/src/components/Nav.tsx

```diff
@@ -29,6 +34,75 @@ const useStyles = makeStyles((theme) =>
       backdropFilter: 'blur(30px)',
       zIndex: theme.zIndex.appBar,
     },
+    banner: {
+      padding: '20px',
+      boxSizing: 'border-box',
+    },
+    bannerDivider: {
+      borderBottom: `1px solid ${theme.palette.divider}`,
+      width: '100%',
+    },
+    bannerContent: {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    bannerText: {
+      fontSize: '15px',
+      fontWeight: 500,
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      maxWidth: 'calc(100% - 80px)', // Adjust based on your needs
+    },
+    readMoreButton: {
+      fontSize: '15px',
+      background: 'none',
+      border: 'none',
+      color: theme.palette.primary.main,
+      padding: 0,
+      font: 'inherit',
+      cursor: 'pointer',
+      outline: 'inherit',
+      marginLeft: '5px',
+      '&:hover': {
+        color: theme.palette.text.secondary,
```

Comment:
do we use this color for hover? it somehow isn't matching up with rest of the theme. can you double-check?